funnel/views/schedule.py

```diff
@@ -36,11 +37,17 @@ def session_data(session, with_modal_url=False, with_delete_url=False):
         {
             'id': session.url_id,
             'title': session.title,
-            # `start` and `end` are legacy
-            'start': session.start_at.isoformat() if session.scheduled else None,
-            'end': session.end_at.isoformat() if session.scheduled else None,
-            'start_at': session.start_at.isoformat() if session.scheduled else None,
-            'end_at': session.end_at.isoformat() if session.scheduled else None,
+            'start_at': (
+                localize_timezone(session.start_at, tz=session.project.timezone)
+                if session.scheduled
+                else None
+            ),
+            'end_at': (
+                localize_timezone(session.end_at, tz=session.project.timezone)
+                if session.scheduled
+                else None
```

Comment:
We have these if conditions in too many places, in both session and project models, so maybe they should be properties: `localized_start_at` and `localized_end_at` (in both models, both returning a localized timestamp or None). @iambibhas

Reply:
Make this a separate issue as there are multiple places to update. There may be another property needed to also render as `isoformat()`, because that too has defensive `if` conditions.